source/SIL.AppBuilder.Portal/src/routes/(authenticated)/tasks/+page.server.ts

```diff
@@ -3,6 +3,10 @@ import type { PageServerLoad } from './$types';
 
 export const load: PageServerLoad = async (event) => {
   const session = await event.locals.auth();
+  //----temporary, remove later-----
```

Comment:
I have that in there so I can get a real productId for testing purposes (since /tasks/[product_id] is the path). I had created a product in S1 locally, but it wasn't showing any available tasks in S2, probably because we haven't implemented workflows yet, so I did this as a temporary measure to be able to test the database queries to get actual data based on the productId.

Reply:
It looks like I also can't access a task in local S1. It shows that the startup workflow has started, but it does not appear to have ever completed. That's probably why I couldn't access see any tasks in S2 either.

Reply:
I found a way to get product id without needing this extra query. Removed.